raw_data/preprocessed/africa/nigeria_english.txt

```diff
@@ -0,0 +1,193 @@
e environmental effects of the project; "mandatory study" means an environmental assessment that is conducted pursuant to section 17 and that includes a consideration of the factors set out in section 11 of this Act; "mandatory study list" means the list in the Schedule to this Act and those that may be prescribed pursuant to section 55[1][c] of this Act; "mandatory study report" means a report of a mandatory study that is prepared in accordance with the provisions of this Act or any regulation made thereunder; "mediation" means an environmental assessment that is conducted with the assistance of a mediator appointed pursuant to section 32 of this Act and that includes a consideration of the factors set out in section 11 of this Act; "mitigation" means, in respect of a project, the elimination, reduction or control of the adverse environmental effects of the project, and includes restitution for any damage to the environment caused by such effects through replacement, restoration, compensation or any other means; "prescribed" means prescribed by regulations; "project" means a physical work that a proponent proposes to construct, operate, modify, decommission, abandon or otherwise carry out or a physical activity that a proponent proposes to undertake or otherwise carry out; "proponent", in respect of a project, means the person, body or Federal authority that proposes the project; "record" includes any correspondence, memorandum, book, plan, map drawing, diagram, pictorial or graphic work, photograph, film, microform, sound recording, videotape, machine readable record, and any other documentary material, regardless of physical form or characteristics, and any copy thereof; "responsible authority" in relation to a project, means a Federal authority that is required pursuant to subsection 14[1] of this Act to ensure that an environmental assessment of the project is conducted; "responsible Minister" means, in respect of a responsible authority: [a] in the case of a department or a ministry in a State, the Minister or Commissioner presiding over that department or ministry; and [b] in any other case, such Minister or Commissioner as the President or Governor may designate as the responsible Minister or Commissioner for that responsible authority; "screening" means an environmental assessment that is conducted pursuant to section 19 of this Act and that includes a consideration of the factors set out in section 17[1] of this Act; "screening report" means a report that summarises the results of a screening. [2] For the purposes of this Act, a company is controlled by another company if: [a] securities of the corporation to which are attached more than fifty percent of the votes that may be cast to elect directors of the corporation are held, other than by way of security only, by or for the benefit of that corporation; and [b] the votes attached to those securities are sufficient, if exercised, to elect a majority of the directors of the corporation.
```

Comment:
Please replace `subsections `11[1] and [2]` with `sections 11[1] and [2]`. Also, please remove the space between `61[1]` and `[b]`. Finally, please replace `subsection `14[1]` with `section 14[1]`.